### ISSUE TITLE
backport exec related api to Classic

### ISSUE THREAD

**Issue by tg123** [kubernetes-client/csharp#889]
exec, port forwarding and other `connect` apis were known work in ver 4.0.26.
should back port related changes back to Classic to make it work with net462+

>did you test if the 3.0.6 websocket working?
>
>if so, we could port it back to current version

I'm not too familiar with WebSockets but it appears to be working perfectly fine in version 4.0.26. The example code worked with `ls`, and I was also able to (for example) change the `ls` command to `new string[] { "touch", "testing.txt }`, and the file appears as expected. This is with net47.

_Originally posted by @cody-malcolm in https://github.com/kubernetes-client/csharp/discussions/887#discussioncomment-2947725_

**Comment by cody-malcolm** [comment]
I'm weighing options on how to deal with the current absence of this functionality.

Could you briefly describe what would be required to backport this functionality if I was to implement the fix and submit a PR?

And if I do not implement the fix and just wait for an existing contributor to do it, how long does a fix of this size generally take?

Thanks!

**Comment by tg123** [comment]
1 or 2 weekends
depends on how busy during weekdays.

a bit busy recent weeks :(


I think I should take a look how 4.0.26 works with websocket certificates and port them back, that is all the work current in my mind

**Comment by tg123** [comment]
@cody-malcolm 
I verified locally
could you please test if the PR fit your scenario?

**Comment by cody-malcolm** [comment]
What method does this version use? My IDE does not recognize the `WebSocketNamespacedPodExecAsync` method.

If this is the method I'm supposed to be using and it should be present, I may not have built/accessed the code correctly. To test the code, I built the Classic .csproj (which recursively built Models and Basic as well), then used `dotnet pack` to create .nupkg files for all three (Classic, Models, Basic). Then I added these .nupkg files to a local feed and used Visual Studio to add the nuget packages (v 8.0.11). I tested that this is working for other methods such as listing pods. Let me know if I also need to build/pack the full release or anything else to get this working.
